pwiz_tools/Skyline/Model/Irt/IrtStandard.cs

```diff
@@ -454,11 +454,9 @@ private static DbIrtPeptide MakePeptide(string sequence, double time)
             return new DbIrtPeptide(new Target(sequence), time, true, TimeSource.peak);
         }
 
-        public static IrtStandard WhichStandard(ICollection<Target> peptides, out HashSet<Target> missingPeptides)
+        public static IrtStandard WhichStandard(ICollection<Target> peptides)
         {
-            var standard = ALL.FirstOrDefault(s => s.ContainsAll(peptides.Select(p => new DbIrtPeptide(p, 0, true, TimeSource.peak)).ToList(), null)) ?? EMPTY;
-            missingPeptides = new HashSet<Target>(standard.Peptides.Where(s => !peptides.Any(p => p.Equals(s.Target))).Select(s => s.Target));
-            return standard;
+            return ALL.FirstOrDefault(s => s.ContainsAll(peptides.Select(p => new DbIrtPeptide(p, 0, true, TimeSource.peak)).ToList(), null)) ?? EMPTY;

```

Comment:
Probably should get:
peptides.Select(p => new DbIrtPeptide(p, 0, true, TimeSource.peak)).ToList()
out of the loop so it only gets called once